lib/exercism/user.rb

```diff
@@ -148,6 +148,16 @@ def invite_to_track_mentor(language)
     update_acls_for_track_mentor(language)
   end
 
+  def public_user_attributes
+    result = {}
+    result[:id] = self.id
+    result[:email] = self.email
```

Comment:
I'd rather not expose the email.

On the one hand, we do get the email from the user's public profile on GitHub, so at least at the time that they signed up, it's public.

On the other hand, if they later made it private on GitHub, we will still have it, and we wouldn't necessarily know that they no longer felt comfortable making that data private.